criu/cgroup.c

```diff
@@ -42,6 +42,11 @@ struct cg_set {
 	struct list_head	ctls;
 };
 
+struct task_id_path{
+	pid_t			id;
+	char		*path;
```

Comment:
as we discussed previously, that is passed with [syscalls](https://github.com/checkpoint-restore/criu/blob/criu-dev/criu/namespaces.c#L1389) so what will happen is you will pass a pointer, not the string. And that pointer will probably be invalid on the other side.